ddtrace/contrib/django/patch.py

```diff
@@ -82,7 +92,14 @@ def cursor(django, pin, func, instance, args, kwargs):
             "django.db.alias": alias,
         }
         pin = Pin(service, tags=tags, tracer=pin.tracer, app=prefix)
-        return dbapi.TracedCursor(func(*args, **kwargs), pin, config.django)
+        cursor = func(*args, **kwargs)
+        traced_cursor_cls = dbapi.TracedCursor
+        if (
+            Psycopg2TracedCursor is not None
+            and isinstance(cursor.cursor, psycopg_cursor_cls)
```

Comment:
do we need to protect `cursor.cursor`? what is `cursor` was `None` or something unexpected?

Reply:
this is a good point, might be worth 🦆-typing